packages/react/src/primitives/shared/utils.ts

```diff
@@ -45,8 +45,8 @@ export type Modifiers = string | number | null;
  */
 export const classNameModifier = (
   base: ComponentClassName,
-  modifier: Modifiers
-): string => {
+  modifier?: Modifiers
+): string | null => {
   return modifier ? `${base}--${modifier}` : null;
```

Comment:
I'm actually think it might make more sense to return empty string here, instead of `null`
```suggestion
  return modifier ? `${base}--${modifier}` : '';
```